go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/spf13/cobra v1.2.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
+	golang.org/x/tools v0.1.10 // indirect
```

Comment:
why this change?

Reply:
It was auto-generated by one of the commands